services/galley/test/integration/API.hs

```diff
@@ -516,6 +517,29 @@ postCryptoMessage4 = do
   postProtoOtrMessage alice (ClientId "172618352518396") conv m
     !!! const 403 === statusCode
 
+-- @SF.Separation @TSFI.RESTfulAPI @S2
+postMessageMissingClients :: TestM ()
+postMessageMissingClients = do
```

Comment:
```suggestion
postMessageRejectIfMissingClients :: TestM ()
postMessageRejectIfMissingClients = do
```

Purpose, Procedure and Expected Result.

Now the name arguably covers Purpose and Expected Result.  I agree that Procedure should be appearent from the code, let's see if we're right.